### PR TITLE
Updated the test as a demo

### LINKED ISSUE #1: Readme needs updating
Readme is sparse.  Include details pls :)

### PR DESCRIPTION
**Pull request by robmarcer**
Fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
 # testingspace
+
+Hello, this is an edit.
+
```
Comment (Steve-Mcl) [inline README.md]:
```suggestion
### Hello, this is an edit.
```

This line would be better as H3 & no new line afterwards.

Comment (robmarcer) [reply README.md]: Thanks @Steve-Mcl, really helpful.

File: README.md [hunk 2]
```diff
@@ -1 +1,4 @@
 # testingspace
+
+### Hello, this is an edit.
+This is a second edit.
```
Comment (Steve-Mcl) [inline README.md]:
```suggestion
### Hello, this is an edit.
This is a second edit containing more detials and inflromatoin.
```

Add extra info

Comment (robmarcer) [reply README.md]:
```suggestion
### Hello, this is an edit.
This is a second edit containing more details and information.
```
Corrected a couple of typos.

Comment (Steve-Mcl) [reply README.md]: 🤦DOH!  Well spotted. 

Happy with your corrections

Comment (robmarcer) [reply README.md]: No problem, thanks.

**Pull request merged**